.github/workflows/roman_ci.yml

```
@@ -48,9 +48,9 @@ jobs:
       cache-key: data-${{ needs.data.outputs.webbpsf_hash }}-${{ needs.crds_contexts.outputs.roman }}
       cache-restore-keys: webbpsf-${{ needs.data.outputs.webbpsf_hash }}
       envs: |
-        - linux: py39-oldestdeps-webbpsf-cov
+        - linux: py310-oldestdeps-webbpsf-cov
           pytest-results-summary: true
-        - linux: py39-webbpsf
+        - linux: py310-webbpsf
           pytest-results-summary: true
         - linux: py310-webbpsf-nolegacypath
```

```suggestion
        - linux: py310-webbpsf-nolegacypath
```
How about we drop the py39 job instead of updating it to 3.10 to leave 3.10 testing to the nolegacy path job?

---

sounds good!